projects/plugins/jetpack/modules/carousel/jetpack-carousel.php

```diff
@@ -606,10 +606,12 @@ class_exists( 'Jetpack_AMP_Support' )
 				! preg_match( '/wp-block-jetpack-slideshow_image/', $image_html ) ) {
 				$attachment_id = absint( $class_id[2] );
 				/**
-				 * If exactly the same image tag is used more than once, overwrite it.
-				 * All identical tags will be replaced later with 'str_replace()'.
+				 * The same image tag may be used more than once but with different attribs,
+				 * so save each of them against the attachment id.
 				 */
-				$selected_images[ $attachment_id  ] = $image_html;
+				if ( ! isset( $selected_images[ $attachment_id ] ) || ! in_array ( $image_html, $selected_images[ $attachment_id ] ) ) {
```

Comment:
PHPCS is complaining that `in_array (` has a gap between the function name and the parenthesis, and that `in_array` isn't providing `true` as its third argument.